_maps/map_files/NSVBlueshift/Blueshift.dmm

```diff
@@ -63666,6 +63666,8 @@
 	c_tag = "Engineering - Secure tech storage";
 	name = "engineering camera"
 	},
+/obj/item/circuitboard/machine/circuit_imprinter/department,
+/obj/item/circuitboard/machine/protolathe/department,
```

Comment:
: )

Reply:
Yeah the map merger marker popped up but given it was such a smol change i just manually did those two rather than doing the direct-code way this time